pkg/output/json.go

```diff
@@ -52,12 +59,41 @@ func newResourceJSON(r *schema.Resource) resourceJSON {
 	}
 }
 
-func ToJSON(resources []*schema.Resource) ([]byte, error) {
+func showSkippedResourcesJSON(resources []*schema.Resource, showDetails bool) []string {
+	unSupportedTypeCount, _, unSupportedCount, _ := terraform.CountSkippedResources(resources)
```

Comment:
"unsupported" is a valid word, so we could use `unsupportedTypeCount` and `unsupportedCount` as the var names. Alternatively we could use `skippedTypeCount` and `skippedCount`.

Reply:
Changed the words, I used the unsupported because in the future we have free resources which are skipped but are unsupported.

Reply:
Just changed the "unsupported" case. I think it's better to stick to skipped/unsupported